Investigate error in `test_cluster_sync`

## Description

During the launch of `Workload benchmarks metrics for 4.10.0 - Beta 1` (https://github.com/wazuh/wazuh/issues/26995), the `test_cluster_sync` produced the following error:

```console
E       AssertionError: 
E         Found 6 times in a row in worker_25: 2024/11/25 12:51:07 INFO: [Worker CLUSTER-Workload_benchmarks_metrics_B798_manager_25] [Integrity sync] Files to create: 0 | Files to update: 1 | Files to delete: 0
...
E         Found 5 times in a row in worker_24: 2024/11/25 12:51:00 INFO: [Worker CLUSTER-Workload_benchmarks_metrics_B798_manager_24] [Integrity sync] Files to create: 0 | Files to update: 1 | Files to delete: 0
E       assert not {'worker_1': {'log': '2024/11/25 12:51:00 INFO: [Worker CLUSTER-Workload_benchmarks_metrics_B798_manager_1] [Integrity..._manager_12] [Integrity sync] Files to create: 0 | Files to update: 1 | Files to delete: 0', 'repeat_counter': 8}, ...}
```
These failures in this test may be related to https://github.com/wazuh/wazuh/issues/27050, as the master node is not functioning correctly

Artifacts : [artifacts.zip](https://github.com/user-attachments/files/17917176/artifacts.zip)

--- a/tests/reliability/test_cluster/test_cluster_logs/test_cluster_sync/test_cluster_sync.py
+++ b/tests/reliability/test_cluster/test_cluster_logs/test_cluster_sync/test_cluster_sync.py
@@ -56,7 +56,7 @@ def test_cluster_sync(artifacts_path):
                                 repeat_counter += 1
                     # If only 1 shared file is synced, it could be the 'client.keys' so it doesn't count as a repeated
                     # log (agents could be registering).
-                    elif sync_logs[i][2:] != (b'0', b'1', b'0', b'0'):
+                    elif sync_logs[i][2:] != (b'0', b'1', b'0'):
                         repeat_counter += 1
 
                     if repeat_counter >= configuration['repeat_threshold']:

